cql3/Cql.g

```diff
@@ -441,30 +441,25 @@ selectClause returns [std::vector<shared_ptr<raw_selector>> expr]
 
 selector returns [shared_ptr<raw_selector> s]
     @init{ shared_ptr<cql3::column_identifier> alias; }
-    : us=unaliasedSelector (K_AS c=ident { alias = c; })? { $s = make_shared<raw_selector>(us, alias); }
-    ;
-
-unaliasedSelector returns [shared_ptr<selectable::raw> s]
-    @init { shared_ptr<selectable::raw> tmp; }
-    :  ( c=cident                                  { tmp = make_shared<selectable::with_expression::raw>(cql3::expr::unresolved_identifier{std::move(c)}); }
-       | K_COUNT '(' countArgument ')'             { tmp = ::make_shared<selectable::with_expression::raw>(make_count_rows_function_expression()); }
-       | K_WRITETIME '(' c=cident ')'              { tmp = make_shared<selectable::with_expression::raw>(
-                                                        cql3::expr::column_mutation_attribute{cql3::expr::column_mutation_attribute::attribute_kind::writetime,
-                                                                                              cql3::expr::unresolved_identifier{std::move(c)}}); }
-       | K_TTL       '(' c=cident ')'              { tmp = make_shared<selectable::with_expression::raw>(
-                                                        cql3::expr::column_mutation_attribute{cql3::expr::column_mutation_attribute::attribute_kind::ttl,
-                                                                                              cql3::expr::unresolved_identifier{std::move(c)}}); }
-       | f=functionName args=selectionFunctionArgs { tmp = ::make_shared<selectable::with_expression::raw>(
-                                                        cql3::expr::function_call{std::move(f), std::move(args)}); }
-       | K_CAST      '(' arg=unaliasedSelector K_AS t=native_type ')'  { tmp = ::make_shared<selectable::with_expression::raw>(
-                                                        cql3::expr::cast{std::move(arg), std::move(t)}); }
+    : us=unaliasedSelector (K_AS c=ident { alias = c; })? { $s = ::make_shared<raw_selector>(us, alias); }
```

Comment:
Unnecessary (?), but not wrong, change?

Reply:
It is necessary. `us` became an std::variant, so argument-dependent lookup now finds std::make_shared since one of the arguments is in the std namespace.